cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsPlatformResources.java

```diff
@@ -395,4 +403,38 @@ public CloudGateWays gateways(CloudCredential cloudCredential, Region region, Ma
     public CloudIpPools publicIpPool(CloudCredential cloudCredential, Region region, Map<String, String> filters) throws Exception {
         return new CloudIpPools();
     }
+
+    @Override
+    public CloudAccessConfigs accessConfigs(CloudCredential cloudCredential, Region region, Map<String, String> filters) throws Exception {
+        CloudAccessConfigs cloudAccessConfigs = new CloudAccessConfigs();
+        AwsCredentialView awsCredentialView = new AwsCredentialView(cloudCredential);
+        AmazonIdentityManagement client = awsClient.createAmazonIdentityManagement(awsCredentialView);
+        try {
+            ListInstanceProfilesResult listRolesResult = client.listInstanceProfiles();
+            for (InstanceProfile instanceProfile : listRolesResult.getInstanceProfiles()) {
+                Map<String, Object> properties = new HashMap<>();
+                properties.put("arn", instanceProfile.getArn());
+                properties.put("creationDate", instanceProfile.getCreateDate().toString());
+
+                cloudAccessConfigs.getCloudAccessConfigs().add(
+                        new CloudAccessConfig(
+                                instanceProfile.getInstanceProfileName(),
+                                instanceProfile.getInstanceProfileId(),
+                                properties));
+            }
+        } catch (AmazonServiceException ase) {
+            if (ase.getStatusCode() == UNAUTHORIZED) {
+                String policyMessage = "Could not get policies on the role because the arn role do not have enough permission: %s";
+                LOGGER.info(String.format(policyMessage, ase.getErrorMessage()));
+                return cloudAccessConfigs;
+            } else {
+                LOGGER.info(ase.getMessage());
+                return cloudAccessConfigs;
+            }
+        } catch (Exception e) {
+            LOGGER.info(e.getMessage());
```

Comment:
This is an unexpected case, I think it shall be logged with error, and also the whole stack trace.